pandas/core/groupby.py

```diff
@@ -1807,6 +1807,13 @@ def get_group_levels(self):
         'ohlc': lambda *args: ['open', 'high', 'low', 'close']
     }
 
+    def _is_builtin_func(self, arg):
```

Comment:
ignore this, was actually an unrelated bug as this wasn't defined on BaseGrouper